apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -31,6 +31,10 @@ const defaultArgs: ModalProps = {
     position: 'center',
     slot: 'This is Lit!',
     dir: 'ltr',
+    leadingAction: {
+        text: 'Confirm',
+        variant: 'primary',
```

Comment:
please add aria label to this object otherwise it isn't clear when visiting storybook that this is a property we can pass in

Reply:
Fixed.